Base64.ts

```diff
@@ -4,19 +4,28 @@ import { TextEncoder } from "./TextEncoder"
 const tables: { [standard in Standard]: string } = {
 	standard: "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/",
 	url: "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_",
+	ordered: "-0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz",
+	reversed: "zyxwvutsrqponmlkjihgfedcba_ZYXWVUTSRQPONMLKJIHGFEDCBA9876543210-",
 }
-export type Standard = "standard" | "url"
+export type Standard = "standard" | "url" | "ordered" | "reversed"
 export function encode(
-	value: ArrayBuffer | Uint8Array | string,
+	value: ArrayBuffer | Uint8Array | string | number | bigint,
 	standard: Standard = "standard",
 	padding: "" | "=" | "-" = ""
 ): string {
-	const data =
-		typeof value == "string"
-			? new TextEncoder().encode(value)
-			: value instanceof Uint8Array
-			? value
-			: new Uint8Array(value)
+	let data: Uint8Array
+	switch (typeof value) {
+		case "string":
+			data = new TextEncoder().encode(value)
+			break
+		case "number":
+		case "bigint":
+			data = new Uint8Array(new BigUint64Array([value as bigint]).buffer)
```

Comment:
You can't cast number to bigint like this; you have to use the BigInt constructor.

See by using a test like 
```
it("reversed timestamp", () => {
		const a = new Date().getTime()
		const b = a + 1
		const c = cryptly.Base64.encode(a as bigint)
		const d = cryptly.Base64.encode(BigInt(b))
		console.log("c: ", c)
		console.log("d: ", d)
	})
```